tensorrt_llm/_torch/cute_dsl_kernels/blackwell/top_k/gvr_topk_decode.py

```
@@ -72,14 +160,12 @@ class GvrParams:
-
-        Mirrors CUDA template specialization GvrParams<T, K>. For K ∈ {512, 1024},
-        cr=1 (DSv3.2) and cr=4 (DSv4, PR #14413) use different kFTarget values
-        — V4's kFTarget=kK alignment eliminates upper-clamp saturation on
-        tight-σ + high-A2 layers; cross-prompt swe-bench shows 1.5-2.2×
-        P2-iter reduction vs V3.2's kFTarget=384/2560. K=2048 is identical
-        across cr (V4 doesn't natively use K=2048; values reused for safety).
+        """Per-(dtype, K, cr) tuning constants, mirroring CUDA's
+        ``GvrParams<T, K>`` template specialization. For K ∈ {512, 1024}
+        cr=1 (DSv3.2) and cr=4 (DSv4, PR #14413) use different kFTarget —
+        V4 aligns kFTarget with kK to avoid upper-clamp saturation on
+        tight-σ layers (1.5-2.2× fewer P2 iters on swe-bench). K=2048 is
```

Ruff (0.15.15)

[warning] Docstring contains ambiguous `σ` (GREEK SMALL LETTER SIGMA). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF002)